app/js/RequestParser.js

```diff
@@ -61,7 +61,7 @@ module.exports = RequestParser = {
       response.imageName = this._parseAttribute(
         'imageName',
         compile.options.imageName,
-        { type: 'string' }
+        { type: 'string', validValues: settings.allowedImageNamesFlat }
```

Comment:
Add a default of settings.clsi.docker.image to handle empty values?

Reply:
I am not sure what you mean here.

_Empty values simply skip the validation logic and I'd rather keep it that way:_

https://github.com/overleaf/clsi/blob/b33734bab64beee8f06327093f9b4bf67c2d5eec/app/js/RequestParser.js#L190-L196

Reply:
You're right, that's not needed.